source/plot/svg/Shapes.ooc

```diff
@@ -1,31 +1,45 @@
 use ooc-math
+use ooc-draw
 
 Shapes: abstract class {
-	line: static func (start, end: FloatPoint2D, lineWidth: Float, opacity: Float, color: String) -> String {
-		"<line x1='" << start x toString() >> "' y1='" & start y toString() >> "' x2='" & end x toString() >> "' y2='" & end y toString() >> "' stroke-width='" & lineWidth toString() >> "' stroke='" >> color >> "' stroke-opacity='" & opacity toString() >> "'/>\n"
+	getColor: static func (bgra: ColorBgra) -> String {
+		"rgb(" clone() & bgra bgr red toString() & "," clone() & bgra bgr green toString() & "," clone() & bgra bgr blue toString() & ")" clone()
```

Comment:
I'm not familiar with String operators, but this one looks interesting. Why do you need to create a String and the immediately call `clone()` on it ?